resteasy/src/test/scala/com/cerner/beadledom/resteasy/ModuleProcessorSpec.scala

```diff
@@ -0,0 +1,84 @@
+package com.cerner.beadledom.resteasy
+
+import com.cerner.beadledom.resteasy.fauxservice.FauxModule
+import org.jboss.resteasy.spi.{Registry, ResteasyProviderFactory}
+import org.scalatest.{FunSpec, MustMatchers}
+import org.scalatest.mockito.MockitoSugar
+import com.google.inject.{AbstractModule, Guice}
+
+class ModuleProcessorSpec extends FunSpec with MockitoSugar with MustMatchers{
+
+  val registry: Registry = mock[Registry]
+  val providerFactory: ResteasyProviderFactory = mock[ResteasyProviderFactory]
+
+  describe("ModuleProcessor") {
+
+    describe("constructor") {
+      it("throws a NullPointerException for null registry") {
+        intercept[NullPointerException] {
+          new ModuleProcessor(null, providerFactory)
+        }
+      }
+
+      it("throws a NullPointerException for null provider factory") {
+        intercept[NullPointerException] {
+          new ModuleProcessor(registry, null)
+        }
+      }
+    }
+
+    describe("ModuleProcessor#processInjector") {
```

Comment:
This can just be "#processInjector"; the output of scala-test will handle formatting based on the nesting.

Reply:
https://github.com/cerner/beadledom/pull/143/commits/baf1e83e23496f193739668eb9632289d1cfbc1c